src/ignore/git.rs

```diff
@@ -313,14 +346,14 @@ mod tests {
         let filter = &result[0];
 
         assert_eq!(filter.regex.as_str(), "C:/Projects/testprj/([^/]+/)*foo");
-        assert_eq!(filter.only_dir, true);
-        assert_eq!(filter.negate, false);
+        assert!(filter.only_dir);
+        assert!(filter.negate);
```

Comment:
Looks like this assertion is the opposite from the original one. Haven't that affected the tests at all?

Reply:
It sure did. I ran `cargo test` on linux, so i didn't catch that since it's a windows test :sweat: